umbraco-cloud/databases/cloud-database.md

```diff
@@ -40,7 +40,7 @@ Now that you've connected you can work with the databases on Umbraco Cloud, like
 LocalDB is no longer supported in the latest major version of Umbraco. The documentation below is only relevant if you are on Umbraco 9 and below.
 {% endhint %}
 
-When you clone a site locally, Umbraco Cloud automatically creates a local database and populates it with data from your website running on the Cloud. If you don't specify anything before starting up your site locally, it'll be a SQL CE database that lives in the `umbraco/Data`folder in Umbraco 9. If you wish to use a local SQL Server instead, you can update the connection string in the web.config or appSettings.json file, but it's important that you do so before your site starts up the first time.
+When you clone a site locally, Umbraco Cloud automatically creates a local database and populates it with data from your website running on the Cloud. If you don't specify settings before local site startup, it defaults to a SQL CE database located in the `umbraco/Data` folder in Umbraco 9. If you wish to use a local SQL Server instead, you can update the connection string in the web.config or appSettings.json file. However, it's important that you do so before your site starts up the first time.
```

Comment:
It is not just normal "settings" it is "database settings" if they are not specified then it will result in the default SQL CE database. 
```suggestion
When you clone a site locally, Umbraco Cloud automatically creates a local database and populates it with data from your website running on the Cloud. If you don't specify database settings before the local site startup, it defaults to a SQL CE database located in the `umbraco/Data` folder. If you wish to use a local SQL Server instead, you can update the connection string in the web.config or appSettings.json file (from v9+). However, it's important that you do so before your site starts up the first time.
```